frontend/src/components/Auth/SignIn.js

```diff
@@ -1,10 +1,6 @@
 import React from 'react';
-// import { Form, Divider, Button } from 'semantic-ui-react';
-// import firebase, { auth } from '../../firebase';
 import { Form } from 'semantic-ui-react';
 import { auth } from '../../firebase';
-// import { uiConfig } from '../../firebase';
-// import StyledFirebaseAuth from 'react-firebaseui/StyledFirebaseAuth';
```

Comment:
Can we leave these line here?

Reply:
Let me bring this code back

Reply:
I will do another pull request and have line 6-7 back